src/odemis/driver/simcam.py

```diff
@@ -207,8 +207,8 @@ def _setTranslation(self, value):
         """
         # compute the min/max of the shift. It's the same as the margin between
         # the centered ROI and the border, taking into account the binning.
-        max_tran = ((self._img.shape[-1] - self._resolution[0] * self._binning[0]) // 2,
-                    (self._img.shape[-2] - self._resolution[1] * self._binning[1]) // 2)
+        max_tran = ((self._img.shape[-2] - self._resolution[0] * self._binning[0]) // 2,
+                    (self._img.shape[-3] - self._resolution[1] * self._binning[1]) // 2)
```

Comment:
I don't think any of this works. The problem is that when the image is RGB, the shape is YXC, while if it's greyscale, the shape is YX. I don't think the code supports passing images with more data (the init code would fail in this case).
The init code already defines a `self._img_res`, which should be the right info (X = img_res[0] and Y =img_res[1]). => Use that attribute.